src/Core/Scheduler/Providers/ThreeDTiles_Provider.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Created On: 2016-11-8
+ * Class: ThreeDTiles_Provider
+ * Description:
+ */
+import * as THREE from 'three';
+import Provider from './Provider';
+import CacheRessource from './CacheRessource';
+import B3dmLoader from '../../../Renderer/ThreeExtended/B3dmLoader';
+import Fetcher from './Fetcher';
+import FeatureMaterial from '../../../Renderer/FeatureMaterial';
+
+
+export function ThreeDTilesIndex(tileset) {
```

Comment:
I think I'd prefer `_3dTilesXXXX` names over, `ThreeDTilesXXX`.

Cons:
   * `_` is supposed to be reserved for private usage
Pros:
  * names that match the procol/spec name
  * _3dTilesIndex is more readable than ThreeDTilesIndex

WDYT?